examples/tensorflow/common/models.py

```diff
@@ -103,6 +96,12 @@ def MobileNetV3(stack_fn, last_point_ch, input_shape=None, model_type='large', *
         name='Conv_2')(x)
     x = activation(x)
 
+    x = layers.GlobalAveragePooling2D()(x)
```

Comment:
These changes will break accuracy for mobilenet_v3. Please, revert these changes.

Reply:
Reverted.